stories/Avatar/Avatar.stories.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { withA11y } from '@storybook/addon-a11y'
+import { withInfo } from '@storybook/addon-info'
+
+import AvatarWithImage from "./AvatarWithImage"
+import AvatarWithText from "./AvatarWithText"
+import AvatarWithIcon from "./AvatarWithIcon"
+import AvatarLink from "./AvatarLink"
+import Esempi from '../Cookiebar/docs/example.md'
+import AvatarIcon from '../../src/components/Avatar/AvatarIcon';
+import AvatarLinkWithTooltip from './AvatarLinkWithTooltip';
+import AvatarSmallList from './AvatarSmallList';
+
+storiesOf('Componenti/Avatar',module)
+    .addDecorator(withA11y)
+    .add(
+        'Avatar with image',
+        withInfo({
+            text: Esempi,
+            propTables: [AvatarWithImage,AvatarIcon]
+        })
+        (()=><AvatarWithImage/>)
```

Comment:
Can you change all these stories from `(()=><AvatarStory/>)` to `(AvatarStory)`?

The problem here is that Storybook only inspects the first component layer, therefore in the Source tab in Storybook it will be shown `<AvatarWithImage>` for instance rather than its content.

Basically now we have this:

<img width="212" alt="Screen Shot 2019-10-23 at 08 54 36" src="https://user-images.githubusercontent.com/924948/67365961-e7782b80-f572-11e9-85d5-416672d13ea6.png">

but would like to have this:

<img width="501" alt="Screen Shot 2019-10-23 at 08 55 26" src="https://user-images.githubusercontent.com/924948/67365970-ed6e0c80-f572-11e9-8f8a-a943c15fd4c0.png">